BasicSample/Assets/Shared/Scripts/InitializeMixedRealityProjectSelectionWindow.cs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+
+using UnityEditor;
+
+namespace Microsoft.MixedReality.OpenXR.BasicSample
+{
+    [InitializeOnLoad]
+    public class InitializeMixedRealityProjectSelectionWindow
+    {
+        static InitializeMixedRealityProjectSelectionWindow()
+        {
+            EditorApplication.update += OnInit;
```

Comment:
We may need a new folder/asmdef for editor-only scripts. I believe this should cause any builds to fail, because the `UnityEditor` namespace doesn't exist in builds.

Reply:
can you elaborate on this Kurtis? I think I addressed it by adding unityeditor namespace to the existing asmdef in this PR if you refer

Reply:
addressed. Moved these to the new "Editor" (old "Build") folder